content/c-sharp/concepts/arrays/terms/resize/resize.md

```diff
@@ -0,0 +1,101 @@
+---
+Title: '.Resize()'
+Description: 'Updates or defines the size of an existing array.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Arrays'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+The **`.Resize()`** method changes the number of elements of a one-dimensional array to the specified new size.
+
+## Syntax
+
+```pseudo
+// Sets a new size for the specified array
+Array.Resize<int>(ref int[] array, int newSize);
+```
+
+`.Resize()` is a modifier of the `Array` object.  When used as part of a one-dimensional array, it defines the number of elements acceptable by the array, or redefines the size of an array.  This can be used to both lengthen or shorten the number of elements within a given array.
+
+## Example
+
+The following example :
+
+```cs
```

Comment:
I see the example is changed from [here](https://learn.microsoft.com/en-us/dotnet/api/system.array.resize?view=net-7.0) but it's still too close. I'd like you to rewrite it more to differentiate it.

Reply:
I have rewritten the example to the point where it cannot possibly be seen as derivative of any other such example.  Whether that makes it a good example or not, I haven't the foggiest, but it's definitely so unique now that it cannot possibly be seen as close to a different example.